client/my-sites/site-settings/composing/index.jsx

```diff
@@ -90,13 +70,22 @@ Composing.defaultProps = {
 };
 
 Composing.propTypes = {
-	onSubmitForm: PropTypes.func.isRequired,
 	handleToggle: PropTypes.func.isRequired,
 	onChangeField: PropTypes.func.isRequired,
+	setFieldValue: PropTypes.func.isRequired,
 	eventTracker: PropTypes.func.isRequired,
 	isSavingSettings: PropTypes.bool,
 	isRequestingSettings: PropTypes.bool,
 	fields: PropTypes.object,
 };
 
-export default localize( Composing );
+export default connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+
+		return {
+			jetpackSettingsUISupported: siteSupportsJetpackSettingsUi( state, siteId ),
+			siteIsJetpack: isJetpackSite( state, siteId ),
```

Comment:
Why backwards? Weren't we using  `props.isJetpackSite` at some other places ?. Well I've just realized out why :P